src/plugins/Workers/TransactionSyncStreamWorker/methods/syncUpToTheGapLimit.js

```diff
@@ -1,5 +1,19 @@
 const logger = require('../../../../logger');
 
+
+async function getBlockHeight(header) {
+  const prevHash = header.prevHash.reverse().toString('hex');
+
+  const prevBlock = await this.transport.getBlockByHash(prevHash);
+  try {
+    const prevBlockHeight = prevBlock.transactions[0].extraPayload.height;
```

Comment:
I've run the tests locally and they produce a bunch of unhandled rejections. This happens because old coinbase transactions don't have `height` in them. While that wouldn't be an issue on the test bench, since it starts a fresh network for every test run, it could be an issue on the mainnet once released. I can't say what would be a proper solution at this point - I'll think about it a little more